src/pipes/safe-html.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({ name: 'safeHtml'})
+export class SafeHtmlPipe implements PipeTransform  {
```

Comment:
@Mohammed9531 We should cover this with its own unit test (I saw that you've added this to the `tree-internal.component.spec.ts`). SafeHtmlPipe should not depend on the tree.